tools/mothballer.py

```diff
@@ -101,14 +102,14 @@ def build_category(categorydir, outdir):
         # open and gather some state
         existing = zipfile.ZipFile(zipfilename, 'r')
         try:
-            category_seed = existing.open(seedfn).read().strip()
-        except:
+            category_seed = existing.open(SEEDFN).read().strip()
+        except Exception:
             pass
         existing.close()
     logging.debug("Using PRNG seed {}".format(category_seed))
 
     zipfileraw = tempfile.NamedTemporaryFile(delete=False)
-    mothball = package(categoryname, categorydir, zfraw)
+    mothball = package(categoryname, categorydir, category_seed)
```

Comment:
This looks like you're switching from a file handle to a random number seed. What?

Reply:
The `package` function's third argument is expected to be a seed value (as bytes), not a file object.

Reply:
That is nuts.